src/Uno.UWP/Storage/Streams/StreamOverBuffer.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace Windows.Storage.Streams
+{
+	internal class StreamOverBuffer : Stream
+	{
+		private readonly Buffer _buffer;
+		private long _position;
+
+		public StreamOverBuffer(Buffer buffer)
+		{
+			_buffer = buffer;
+		}
+
+		/// <inheritdoc />
+		public override bool CanRead { get; } = true;
+
+		/// <inheritdoc />
+		public override bool CanSeek { get; } = true;
+
+		/// <inheritdoc />
+		public override bool CanWrite { get; } = true;
+
+		/// <inheritdoc />
+		public override long Length => _buffer.Length;
+
+		/// <inheritdoc />
+		public override long Position
+		{
+			get => _position;
+			set
+			{
+				if (value < 0) throw new ArgumentOutOfRangeException(nameof(value), "Position cannot be lower than 0");
+				if (value > Length) throw new ArgumentOutOfRangeException(nameof(value), "Position cannot be equals or greater than Length");
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)